src/Polytopes/Polyhedron/properties.jl

```diff
@@ -3,17 +3,30 @@
 ### Iterators
 ###############################################################################
 ###############################################################################
+"""
+    Polyhedra
+
+Dummy type used for specifying the desired output format.
+Each polyhedron is given by a `Polyhedron`.
```

Comment:
Huh, not sure what this is supposed to tell me. Perhaps this?
```suggestion
If this output format is chosen, then polyhedra shall be returned as instances of type `Polyhedron`.
```

Also, whatever the alternative is/are should IMHO be referenced here, perhaps like this: "Alternative output formats for polyhedra are FOO and BAR."